nussknacker-dist/src/universal/bin/run.sh

```diff
@@ -44,6 +48,10 @@ if [[ "${RUN_IN_BACKGROUND}" == "true" ]]; then
   echo $! > $PID_FILE
   echo "Nussknacker up and running"
 else
+  echo "JVM:"
```

Comment:
maybe sth like ```echo "JVM: `java --version`"``` (--version is to stdout, -version to stderr :) )

Reply:
ok